styles/template2.css

```diff
@@ -1,3 +1,158 @@
+.home-page {
+  height: 100vh;
+}
+
+#nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+.internal-link {
+  display: none;
+}
+
+@media screen and (min-width: 750px) {
+  #nav {
+    margin-top: 0;
+    justify-content: center;
+  }
+
+  header #nav .logo {
+    margin-right: 30%;
+    font-family: Poppins, sans-serif;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  #nav nav {
+    display: none;
+  }
+
+  #nav .internal-link {
+    display: flex;
+    justify-content: space-between;
+    min-width: 190px;
+  }
+
+  #nav .internal-link li {
+    list-style: none;
+  }
+
+  #nav .internal-link li a {
+    text-decoration: none;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+}
+
+#home_presentation {
+  background-image:
+    url("../statics/images/Ellipse 11.png"),
+    url("../statics/images/Rectangle 4.png"),
+    url("../statics/images/Ellipse\ 3.png"),
+    url("../statics/images/Ellipse\ 8.png"),
+    url("../statics/images/Ellipse\ 5.png"),
+    url("../statics/images/Rectangle\ 7.png"),
+    url("../statics/images/Ellipse 12.png"),
+    url("../statics/images/Ellipse 4.png"),
+    url("../statics/images/Rectangle\ 5.png"),
+    url("../statics/images/Ellipse\ 13.png"),
+    url("../statics/images/Ellipse\ 15.png");
+  background-position:
+    19px 79px,
+    294px 68px,
+    323px 102px,
+    214px 110px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
+
+@media screen and (min-width: 750px) {
```

Comment:
- [ ] For the Desktop breakpoint the value for `min-width` property should be in range of `992px - 1100px` your break point is 750px kindly fix the issue.

![image](https://user-images.githubusercontent.com/43698511/134555217-dd07d45c-3e10-4ef0-9f03-ed5076512203.png)
original design :
![image](https://user-images.githubusercontent.com/43698511/134555301-16c1c967-e105-4692-afd6-04016171f40a.png)

- [ ] As you can see from the pictures for the desktop view the headline section looks a bit different from the one in the original design. The background image doesn't fit the whole screen. Kindly fix the issue.

![image](https://user-images.githubusercontent.com/43698511/134557638-69d3f02b-45dc-4841-82f3-ea305ce2a756.png)
original design:
![image](https://user-images.githubusercontent.com/43698511/134555956-d0958518-2e77-48ac-a7e0-c50f9a8a66e3.png)

- [ ] The project's cards also look a bit different from the original design. I think you need to adjust the margins and center the cards. Kindly fix the issue.

original design : 
![image](https://user-images.githubusercontent.com/43698511/134556815-c1618bf1-411f-45a3-88ea-d4ba723045e3.png)
your webpage:
![image](https://user-images.githubusercontent.com/43698511/134556873-e1c4fade-e9d0-49b3-bb63-196b004a4fc7.png)

- [ ] The about me section also looks a bit different. It has 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width. The elements of each bloc are aligned to the right. The languages cards and the borders for the technologies look also different Kindly fix the issue.

![image](https://user-images.githubusercontent.com/43698511/134557451-8f07d9d0-367d-4276-9cf6-2b6e63052fb9.png)
![image](https://user-images.githubusercontent.com/43698511/134558107-c4e40912-d890-474a-9908-96c1fdc4a1e9.png)
mobile view:
![image](https://user-images.githubusercontent.com/43698511/134558281-64106718-4752-4925-babf-0441e8c30ab2.png)

original design:
![image](https://user-images.githubusercontent.com/43698511/134557772-6c0d3129-8f4d-4553-ba0d-48d6d48e9d68.png)
mobile view:
![image](https://user-images.githubusercontent.com/43698511/134558385-4408dcfd-4037-43f9-b944-08d300e53506.png)

- [ ] Kindly fix the background image of the contact-me section for desktop and mobile view.